tests/unittest/_torch/multi_gpu_modeling/test_llama4.py

```diff
@@ -18,7 +18,15 @@
 @pytest.mark.parametrize("tp_size", [8], ids=["tp8"])
 @pytest.mark.parametrize("use_cuda_graph", [True, False],
                          ids=["enable_graph", "disable_graph"])
-def test_llama4(model_name, backend, tp_size, use_cuda_graph):
+@pytest.mark.parametrize("ep_size", [4, 1], ids=["ep4", "ep1"])
+@pytest.mark.parametrize("pp_size", [1, 8], ids=["pp1", "pp8"])
+def test_llama4(model_name, backend, tp_size, use_cuda_graph, ep_size, pp_size):
+    if pp_size > 1 and (ep_size > 1 or tp_size > 1):
```

Comment:
The tp_size is always 8. Will the pp8 test actually run?

Reply:
My bad. Meant to add one more case